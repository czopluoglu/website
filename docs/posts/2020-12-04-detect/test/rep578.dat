00010000011110101110
01111100001101011101
11111111110110111111
00010001001110100110
10010011111011010110
01100001011110101111
11001110111100000111
01000001101011000011
11100111111010111110
11010111011110000110
00111011101111111110
01001101111101100110
00010011010110100011
11011001110011000111
10000000010000101110
00001010001010110011
01001000001001000111
00011001011010000111
10110110111000000011
10111011010010110010
11110101111010111101
01111001110100100001
10110001111110101011
01000101011010001010
10011011000111100111
11111101100100100111
01111110110010110011
10101001100010000000
00101111101100101110
11110111111000111111
10011101111011011110
10111110111111111110
10011000101011101111
11001101011001000111
10101101011110111111
01001000001010101111
11001000111111101111
01011110111111011110
10111011010010100010
10011100011010101110
10111011000110101111
11110111100111110111
10111010110001001110
11010011111111111111
00010000011111101111
00001111111110110110
10101000010110101111
00000000111010111011
01100101011111011101
01000111111111111111
10001000001101111110
00100010111000001011
00001101110001100011
10110110110110001011
11110010010010100110
00100111101011100011
10000011011010100111
11111101111110101110
11111111111110111110
00101100001010101010
01100011100010110000
00010010001011000110
00110101111011111111
01000111001001100011
00110011011010001110
11010001001100010111
10001011010011011001
10101100101001001110
01111111010001000010
11000011011110000101
11010111101110111110
10010001101110111111
10010011110010110000
10011100100000100011
10001001111010010101
01110011100000000010
11110110110111100100
01000011111001110110
00101101001010000010
10100010111110110100
10000000001110001111
00001100001101100010
01000000000100000110
11111010111111111111
00000001001000100111
10011011101001000010
11100001011100100000
01100001001110001111
10000100101100111111
01011001100010111111
01100001000110110010
10000011010000100110
10100000011111111000
01110000001111011111
01011101110001100010
01000111010011101001
01000000100010010001
00110001111110000110
10011001100010100110
11001001110110111110
10010001101101110111
00001000000001000111
00011000100101101111
00000000111110111110
00100100011000101010
00100011001111110111
11101010111011011110
00000011101010000010
00001101101101111010
00001001000111100110
01000101100001000111
00111111011100111110
00110000101011011111
10110101111010110010
10011111111011000011
00101000011010100011
00001010001101011010
01001011101010101010
01000100111011100101
00100001011111101111
01110111110010101010
10111111111101101110
11110111111001000110
11111101100011100000
10001101111000110110
11101001001111101101
11101111110010101111
10000000001000110011
10111110011111111111
10111100011001111000
10111101111000000011
11111110011110110111
10001000001101000010
11011100111100111010
00011000000111101111
00000000001011101110
10001011011110000110
10001010001110000100
10111101011101111111
11111111111011100111
10101111110110100010
01001010011000001011
10000111110000000011
00010100100011100010
10001111111111110111
10001101011000011111
00100010001011010110
01100001110110111001
11010000111010110101
11001100011111001111
01011101011100001110
00001100000011010010
10100000011010111111
01101101111011111110
01000010111000100000
00010000010101001101
01001001001111111110
01011101111011000011
00101111001001000011
11001111111111011111
01110011000011100111
10000100111000011101
11001001011111000111
10000001011000111011
01001000000001000000
01000000000101101111
11111011010110101010
01000101001110011110
10100000011110011010
00000000000000000010
11100111010111111111
10101110111100001110
00110001011010001100
01000000111111101111
10001001001001011111
00110101110010001101
10011110110110010111
10000001101001000000
10001101101011011010
10101100001000110110
11110111110001101101
10100001011011001010
01000100001000101010
10010100111110001111
01001011001010000111
11100111110110000101
00000001001110100110
11111111111111101111
10010000111110100111
11101101011001010101
01010000001111101011
01111101101111001010
00000000101110110101
10001101111001001000
11111111111011111111
01010100001111000110
10111000111000001101
00100010110000000110
01010101000001110111
00100000011110101001
00001100000010110010
10001001010010100100
11011111111001111011
11000100011000110010
01100001011111111000
11110111111011111111
11111111110010100011
10011001000110000110
10100001001010010110
00000101101010100110
10011000111000101011
11000101001011101110
11110111111111011111
10101110001110000011
11101011111111111111
11000101111101111000
11010100101100111001
11111101111111111111
10100011101110101000
00100111011110011111
00001101111011110011
01011001110110100010
10110111001010000111
01101001001010111111
11000000001111001110
11010110011111111111
01101011101101011110
10110011101100000010
11010001101010000001
11111111110011101001
01101100001000110110
01001001001111000100
01100000101000011110
11100101010111111111
00000001111110010100
11100011111001111111
01011101101000101111
01000101111100010010
00011001110010100111
10100101000111001110
10010101101011110111
00001000101010111111
01000101100110001000
01101001001000101101
01100011111111011011
00000011100101000011
11001001111011000100
01100111111001110011
10011010000111111111
11110101111011111111
