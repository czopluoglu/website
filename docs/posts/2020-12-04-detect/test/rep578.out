-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep578.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.96
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   1.0225
                   IDN index value:   0.7526
                           Ratio r:   0.7001

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    1    1    3    1    1    1    1    1
    4    4    4    2    5    4    4    4    3    2

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    3    4    6    7    8    9   10

   -----------CLUSTER  2-------------

    2   14   20

   -----------CLUSTER  3-------------

    5   19

   -----------CLUSTER  4-------------

   11   12   13   16   17   18

   -----------CLUSTER  5-------------

   15
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++++-++---+------+-
   +d++++-+-------+-+--
   ++d++-++--+++--+---+
   +++d-+++++-+++------
   +++-d+++----+--+----
   -+-++d+++------+----
   +-++++d++-------+---
   +++++++d--+------+--
   ---+-++-d++---+----+
   ---+----+d+++++-+++-
   --+----+++d------+++
   +-++-----+-d+-------
   --+++----+-+d+----++
   ---+-----+--+d+-+++-
   --------++---+d+++++
   -++-++--------+d+-++
   ------+--+---+++d+++
   -+-----+-++--++-+d++
   +--------++-++++++d-
   --+-----+-+-+-++++-d
 
 No cross validation for this DETECT run
